### DISCUSSION TITLE
Can't Reapply - Rejecting the Validity of my Campus Email

### DISCUSSION THREAD

**Discussion by yhafez** [community/community#46548] [GitHub Education]
Hello,

I'm currently enrolled as domestic distance-learner in the M.S. in Computer Science and Information Systems at the University of Michigan-Flint. I was previously affiliated with the University of Michigan-Ann Arbor campus, so rather than issue me a new UM Flint specific e-mail, my UM-wide e-mail was re-activated. I was initially rejected from GitHub Education on the premise that I'm 1) not near my campus, however I'm a domestic distance learner and 2) my e-mail isn't associated with my campus. When I go to re-apply and select my UM-wide e-mail, which is the only e-mail I have associated with my university system, I get an error message telling me that only UM-Flint email addresses can be used and the button to submit my application request is disabled, preventing me from reapplying.

I tried contacting GitHub for support on this and my ticket was closed without any investigation. Please advise.

Thank you

**Comment by vandana123maddi** [comment]
Yes me to i got rejected for not near my campus i can't understand what they mean  i was near to my campus  can any one  say at what bases they will judge ur near to compus or not 2) in my college they  won't provide any college student  Gmail
